src/cli.rs

```diff
@@ -223,19 +223,11 @@ pub struct Img {
     #[arg(long, env = "SWWW_TRANSITION_STEP", default_value = "10")]
     pub transition_step: u8,
 
-    ///How fast the transition 'sweeps' through the screen
+    ///How long the transition takes to complete in seconds.
     ///
-    ///For any transition except 'simple', they work by starting at some point (or line), and
-    ///working their way through the screen, while changing the colors of the values they've passed
-    ///through. For exemple, 'left' will cause the transition to start from the left of the screen,
-    ///moving towards the right. Every value that's to the left will be updated accordingly. This
-    ///controls how fast the 'going to the right' motion goes.
-    ///
-    ///A value of 'n' means that we'll go 'n' columns at a time for the 'left' transition, for
-    ///example. For the transitions that work with radii ('center', 'outer' and 'any') a value of
-    ///'n' means that we'll increase the radius by 'n' every time.
-    #[arg(long, env = "SWWW_TRANSITION_SPEED", default_value = "5")]
-    pub transition_speed: u8,
+    ///Note that this doesnt work with the 'simple' transition
+    #[arg(long, env = "SWWW_TRANSITION_DURATION", default_value = "3")]
+    pub transition_duration: f32,
```

Comment:
Specify the unit in the description (e.g.: `in seconds`).

Reply:
i did add it, its on the line right above this snippet